.github/workflows/build-release-pip-package.yml

```diff
@@ -21,7 +17,7 @@ jobs:
     - name: Checkout toolkit repository
       uses: actions/checkout@v4
       with:
-        ref: ${{ github.event.inputs.branch }}
+        ref: main
```

Comment:
Here we should checkout the code that will be tagged in the branch (I think it is event.release.target_commitish).